packages/ariakit/src/checkbox/__examples__/checkbox-controlled/index.tsx

```diff
@@ -0,0 +1,17 @@
+import { useState } from "react";
+import { Checkbox } from "ariakit/checkbox";
+import "./style.css";
+
+export default function Example() {
+  const [checked, setChecked] = useState(true);
+  return (
+    <label className="label">
+      <Checkbox
+        className="checkbox"
+        checked={checked}
+        onChange={(e) => setChecked(e.target.checked)}
```

Comment:
Let's be more explicit since it's going to be part of the documentation. I'll see if there's an eslint rule for this.
```suggestion
        onChange={(event) => setChecked(event.target.checked)}
```